src/annotator/highlighter/overlay-highlighter/overlay-utils.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+
+module.exports = {
+
+  createSVGElement: (name) => {
+    return document.createElementNS('http://www.w3.org/2000/svg', name);
+  },
+
+  getCoords: (el) => {
```

Comment:
I take it these are document-relative coordinates, and there is an assumption that the highlight elements are positioned relative to the body.

I wonder if you'll encounter any of the same issues I saw with documents that reposition the `<body>` or `<html>` elements in efbdc813b2825449cba71e33e4fc92cd0eec5685 (see #487 )